subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolveengine/graph/ComponentResult.java

```diff
@@ -38,4 +39,8 @@
      * Returns a unique id for this component.
      */
     ComponentIdentifier getComponentId();
+
+    String getVariantName();
```

Comment:
:x: Let's add javadoc to our internal APIs

Reply:
Added